### ISSUE TITLE
Unable to switch page 

### ISSUE THREAD

**Issue by ggoodquestion** [KwabenBerko/News-API-Java#3]
This might be a bug for switch page, I read the source code of `TopHeadlinesRequest.class` and found the below
```
public TopHeadlinesRequest.Builder page(int page) {
        this.page = String.valueOf(this.pageSize);
        return this;
}
```
I think  `this.pageSize` should be `page`.

**Comment by KwabenBerko** [comment]
Hi @ggoodquestion 
Sorry about that. Can you kindly create a PR for this?

**Comment by KwabenBerko** [comment]
@ggoodquestion This has been fixed in version `1.0.2`
